tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,105 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
```

Comment:
[Documentation] missing. Please, explain what this test is trying to verify

Reply:
Done